osu.Game.Rulesets.Osu/Edit/OsuSelectionHandler.cs

```diff
@@ -10,7 +15,125 @@ namespace osu.Game.Rulesets.Osu.Edit
 {
     public class OsuSelectionHandler : SelectionHandler
     {
-        public override bool HandleMovement(MoveSelectionEvent moveEvent)
+        public override ComposeSelectionBox CreateSelectionBox()
+            => new ComposeSelectionBox
+            {
+                CanRotate = true,
+                CanScaleX = true,
+                CanScaleY = true,
```

Comment:
These should be off if only one object is selected.

* If only a slider is selected, it doesn't break but behaves funky (grabbing right and lower handles does nothing, while grabbing left and upper handles can move the object around due to the way this is implemented).
* Worse yet trying to scale a single hitcircle crashes to desktop because of division-by-zero reasons.

Reply:
yep, already noted in my head.